request/http_retransport.go

```diff
@@ -1,41 +1,102 @@
 package request
 
 import (
+	"io"
+	"log"
 	"net/http"
 	"sync"
 
 	"github.com/NordSecurity/nordvpn-linux/events"
+	"github.com/NordSecurity/nordvpn-linux/internal"
 )
 
-// HTTPReTransport is std RoundTripper enhanced to reconnect after vpn connect is done
+// HTTPReTransport is modified/enhanced RoundTripper.
 // Thread safe.
 type HTTPReTransport struct {
-	inner    http.RoundTripper
-	createFn func() http.RoundTripper
-	mu       sync.RWMutex
+	// for protects the access to inner and to shouldRecreate. This will allow multiple
```

Comment:
Shouldn't this be a bit rephrased?

IIUC, the `shouldRecreate` refers to the overall logic of recreation an HTTP request. If that's the case then what about:

```suggestion
	// protects access to inner and to recreation logic. This will allow multiple
```